lib/applications/resolvers.ts

```diff
@@ -0,0 +1,145 @@
+import { ApolloError } from 'apollo-server-errors'; // Apollo error
+import { Resolver } from '@lib/resolvers'; // Resolver type
+import {
+  ShopifyConfirmationNumberAlreadyExistsError,
+  ApplicantIdDoesNotExistError,
+} from '@lib/applications/errors'; // Employee errors
+import { DBErrorCode } from '@lib/db/errors'; // Database errors
+
+/**
+ * Query all the RCD applications in the internal-facing app
+ * @returns All RCD applications
+ */
+export const applications: Resolver = async (_parent, _args, { prisma }) => {
+  const applications = await prisma.application.findMany();
+  return applications;
+};
+
+/**
+ * Create an RCD application
+ * @returns Status of operation (ok, error)
+ */
+export const createApplication: Resolver = async (_, args, { prisma }) => {
+  const {
+    input: {
+      firstName,
+      middleName,
+      lastName,
+      dateOfBirth,
+      gender,
+      email,
+      phone,
+      province,
+      city,
+      address,
+      postalCode,
+      notes,
+      rcdUserId,
+      isRenewal,
+      poaFormUrl,
+      applicantId,
+      disability,
+      affectsMobility,
+      mobilityAidRequired,
+      cannotWalk100m,
+      aid,
+      physicianName,
+      physicianMspNumber,
+      physicianAddress,
+      physicianCity,
+      physicianProvince,
+      physicianPostalCode,
+      physicianPhone,
+      physicianNotes,
+      processingFee,
+      donationAmount,
+      paymentMethod,
+      shopifyConfirmationNumber,
+      guardianFirstName,
+      guardianMiddleName,
+      guardianLastName,
+      guardianPhone,
+      guardianProvince,
+      guardianCity,
+      guardianAddress,
+      guardianPostalCode,
+      guardianRelationship,
+      guardianNotes,
```

Comment:
If anyone is nifty with typescript and can suggest an alternative to this monstrosity that would be great 🥺